### DISCUSSION TITLE
Upgrade guideline

### DISCUSSION THREAD

**Discussion by fatihyildizhan** [PrismLibrary/Prism#2263] [WPF]
Hi,

We want to upgrade our WPF project. It is written with Prism 7.2. We want to upgrade to Prism 8. There are many new things and we don't want to break things.

Is there any upgrade guideline?

Thanks

**Comment by brianlagunas** [comment]
Read the release notes https://github.com/PrismLibrary/Prism/releases/tag/v8.0.0.1909
